tmdb/school_registration_validator.py

```diff
@@ -6,8 +6,10 @@ class SchoolRegistrationValidator:
     def __init__(self, school_registration, extracted_data):
         self.extracted_data = extracted_data
         self.school_registration = school_registration
-        self.competitors_by_name = {c['name']:c
-                for c in self.extracted_data.extracted_competitors}
+        self.competitors_by_name = {}
+        for c in self.extracted_data.extracted_competitors:
```

Comment:
This looks fine to me. 
I probably would have been lazy and have done something like: 
`self.competitors_by_name = {c['name'].strip():c
                for c in self.extracted_data.extracted_competitors}`

Oh wait, I guess we probably want the actual value of `name` to also be stripped. Just going to comment with my random thoughts.